arangod/Cluster/ClusterComm.h

```diff
@@ -688,7 +689,7 @@ class ClusterCommThread : public Thread {
  public:
   ClusterCommThread();
   ~ClusterCommThread();
-
+  void haltThreads();
```

Comment:
the name is a bit misleading, because the function will halt just one thread. So what about renaming it to just `halt` or `shutdown`?